test/image/mocks/polar_bar-overlay.json

```diff
@@ -6,12 +6,15 @@
     "opacity": 0.7
   }, {
     "type": "barpolar",
-    "r": [1, 2, 3],
-    "theta": [180, 90, 360],
-    "opacity": 0.7
+    "r": [3, 3],
+    "theta": [0, 45],
+    "opacity": 0.7,
+    "marker": {
+      "line": {"width": 5}
```

Comment:
which looks like

![image](https://user-images.githubusercontent.com/6675409/45059971-087a7600-b06c-11e8-8dfa-d7855ada6fd5.png)

closeup

![image](https://user-images.githubusercontent.com/6675409/45059983-0e705700-b06c-11e8-92a8-b1953a69dc6b.png)

with the default `stroke-mitelimit`.